composer.json

```diff
@@ -28,7 +28,8 @@
         "symfony/yaml": ">=4.4 <6.0",
         "symfony/css-selector": ">=4.4 <6.0",
         "behat/gherkin": "^4.6.2",
-        "codeception/stub": "^3.7"
+        "codeception/stub": "^3.7",
+        "nunomaduro/collision": "^5.1"
```

Comment:
Codeception 5 supports Symfony 4.4 as a minimum, but this collision release requires `symfony/console ^5.0` as a minimum.

Reply:
This is a very good point. In order to support Symfony 4.4 we would have to use nunomaduro/collision v3, which is [not supported anymore](https://github.com/nunomaduro/collision/pull/173).
I think that it is more important to keep support for Symfony 4.4.